ogc/display-ogc-api-collection/src/main/java/com/esri/samples/display_ogc_api_collection/DisplayOgcApiCollectionSample.java

```diff
@@ -0,0 +1,169 @@
+
+      // set title, size, and add JavaFX scene to stage
+      stage.setTitle("Display OGC API Collection");
+      stage.setWidth(800);
+      stage.setHeight(700);
+      stage.setScene(scene);
+      stage.show();
+
+      // authentication with an API key or named user is required to access basemaps and other location services
+      String yourAPIKey = System.getProperty("apiKey");
+      ArcGISRuntimeEnvironment.setApiKey(yourAPIKey);
+
+      // create a map with the standard imagery basemap style
+      ArcGISMap map = new ArcGISMap(BasemapStyle.ARCGIS_TOPOGRAPHIC);
+
+      // create a map view and set the map to it
+      mapView = new MapView();
+      mapView.setMap(map);
+
+      // create a progress indicator
+      var progressIndicator = new ProgressIndicator();
+
+      // add the map view to the stack pane
+      stackPane.getChildren().addAll(mapView, progressIndicator);
+
+      // define strings for the service URL and collection id
+      // note that the service defines the collection id which can be accessed via OgcFeatureCollectionInfo.getCollectionId().
+      var serviceUrl = "https://demo.ldproxy.net/daraa";
+      var collectionId = "TransportationGroundCrv";
+
+      // create an OGC feature collection table from the service url and collection id
+      ogcFeatureCollectionTable = new OgcFeatureCollectionTable(serviceUrl, collectionId);
+
+      // set the feature request mode to manual (only manual is currently supported).
+      // in this mode, the table must be manually populated - panning and zooming won't request features automatically
+      ogcFeatureCollectionTable.setFeatureRequestMode(ServiceFeatureTable.FeatureRequestMode.MANUAL_CACHE);
+
+      ogcFeatureCollectionTable.addDoneLoadingListener(() -> {
+        if (ogcFeatureCollectionTable.getLoadStatus() == LoadStatus.LOADED) {
+
+          // create a feature layer and set a renderer to it to visualize the OGC API features
+          var featureLayer = new FeatureLayer(ogcFeatureCollectionTable);
+          var simpleRenderer = new SimpleRenderer(new SimpleLineSymbol(SimpleLineSymbol.Style.SOLID, ColorUtil.colorToArgb(Color.BLUE), 3));
+          featureLayer.setRenderer(simpleRenderer);
+
+          // add the layer to the map
+          map.getOperationalLayers().add(featureLayer);
+
+          // zoom to a small area within the dataset by default
+          Envelope datasetExtent = ogcFeatureCollectionTable.getExtent();
+          if (datasetExtent != null && !datasetExtent.isEmpty()) {
+            mapView.setViewpointGeometryAsync(
+              new Envelope(datasetExtent.getCenter(), datasetExtent.getWidth() / 3, datasetExtent.getHeight() / 3));
+          }
+
+        } else {
+          // show an alert dialog if there is a loading failure
+          new Alert(Alert.AlertType.ERROR, "Failed to load OGC Feature Collection Table");
+        }
+      });
+
+      // load the table
+      ogcFeatureCollectionTable.loadAsync();
```

Comment:
Think other samples generally put the `loadAsync` before the listener. I find that easier to follow anyway.